plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1163,6 +1164,60 @@ def accept_cisco_ise_server_certificate(self, ipAddress, trusted_server):
             return None
         return
 
+    def check_auth_server_response_status(self, response, validation_string_set):
+        """
+        Format the parameter of the payload for updating the authentication and policy server
```

Comment:
Can you please check the description? It looks like you are checking the task status..

    Checks the status of a task related to updating the authentication and policy server
    by polling the task details until it completes or a timeout is reached.